lib/commons/dom/is-visible.js

```diff
@@ -156,11 +156,15 @@ function isVisible(el, screenReader, recursed) {
   }
 
   // hidden from visual users
+  const elHeight = parseInt(style.getPropertyValue('height'));
   if (
     !screenReader &&
     (isClipped(style) ||
       style.getPropertyValue('opacity') === '0' ||
-      (getScroll(el) && parseInt(style.getPropertyValue('height')) === 0))
+      (getScroll(el) && elHeight === 0) ||
```

Comment:
Is there a reason you left this at 0?

Reply:
Yes, turns out it wasn't relevant to the case I was trying to address so I left it alone. Would you rather it be bumped up too?